src/mongooseHelper/mongoose.service.ts

```diff
@@ -1,4 +1,33 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable, OnApplicationBootstrap } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { Role } from 'src/roles/role.enum';
+import { UserService } from 'src/user/user.service';
 
 @Injectable()
-export class MongooseHelperService {}
+export class MongooseHelperService implements OnApplicationBootstrap {
+  FIRST_ADMIN_EMAIL = 'admin@cinema.com';
```

Comment:
It's better to specify root admin's credentials in the .env file, read them ino the config and use that config while runtime.